runtime/src/main/java/org/corfudb/runtime/collections/DiskBackedCorfuTable.java

```diff
@@ -135,13 +136,15 @@ public DiskBackedCorfuTable(@NonNull PersistenceOptions persistenceOptions,
                                 @Nonnull Index.Registry<K, V> indices) {
 
         this.persistenceOptions = persistenceOptions;
-        this.secondaryIndexesAliasToPath = new HashMap<>();
+        this.secondaryIndexesPath = new HashMap<>();
         this.indexToId = new HashMap<>();
         this.indexSpec = new HashSet<>();
 
         byte indexId = 0;
         for (Index.Spec<K, V, ?> index : indices) {
-            this.secondaryIndexesAliasToPath.put(index.getAlias().get(), index.getName().get());
+            this.secondaryIndexesPath.put(index.getAlias().get(), index.getName().get());
```

Comment:
should these be under empty checks - like we dont want to populate the alias if one isn't defined?